.github/workflows/automerge.yml

```
@@ -0,0 +1,60 @@
+name: automerge
+on:
+  schedule:
+    - cron: '4 5 * * *'  # Runs at 04:05 (random) UTC every day
+  pull_request:
+    types:
+      - labeled
+      - unlabeled
+      - synchronize
+      - opened
+      - edited
+      - ready_for_review
+      - reopened
+      - unlocked
+  pull_request_review:
+    types:
+      - submitted
+      - dismissed
+  check_suite:
+    types:
+      - completed
+  status: {}
+jobs:
+  automerge:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0  # Ensures all history is fetched
+
+      - name: Get date of the last commit
+        id: last_commit_date
+        run: |
+          LAST_COMMIT_DATE=$(git log -1 --format=%cI)
+          echo "Last commit date: $LAST_COMMIT_DATE"
+          echo "::set-output name=last_commit_date::$LAST_COMMIT_DATE"
```

```suggestion
          echo "last_commit_date=$LAST_COMMIT_DATE" >> "$GITHUB_OUTPUT"
```

The `::set-output` method is deprecated; you need to write to `$GITHUB_OUTPUT` now.